src/VirtualClient/VirtualClient.Actions.UnitTests/MLPerf/MLPerfMetricsParserTests.cs

```diff
@@ -27,60 +27,67 @@ private string ExamplePath
         }
 
         [Test]
-        [TestCase("Example_bert_accuracy_summary2.json")]
-        [TestCase("Example_bert_perf_harness_summary2.json")]
-        public void MLPerfParserHandlesFilesThatHasNoValidValues(string exampleFile)
+        [TestCase("Example_performance_summary1.json")]
+        public void MLPerfParserVerifyValidPerformanceMetrics(string exampleFile)
         {
             string outputPath = Path.Combine(this.ExamplePath, exampleFile);
             this.rawText = File.ReadAllText(outputPath);
 
-            if (exampleFile.Contains("accuracy"))
-                this.testParser = new MLPerfMetricsParser(this.rawText, true);
-            else
-                this.testParser = new MLPerfMetricsParser(this.rawText, false);
+            this.testParser = new MLPerfMetricsParser(this.rawText, false);
+            IList<Metric> metrics = this.testParser.Parse();
+
+            Assert.AreEqual(2, metrics.Count);
+            MetricAssert.Exists(metrics, "DGX-A100_A100-SXM4-40GBx8_TRT-custom_k_99_MaxP-Server-PerformanceMode", 1, "VALID/INVALID");
+            MetricAssert.Exists(metrics, "DGX-A100_A100-SXM4-40GBx8_TRT-custom_k_99_MaxP-Server-result_completed_samples_per_sec", 25405.6);
+        }
 
+        [Test]
+        [TestCase("Example_performance_summary2.json")]
+        public void MLPerfParserVerifyInvalidPerformanceMetrics(string exampleFile)
+        {
+            string outputPath = Path.Combine(this.ExamplePath, exampleFile);
+            this.rawText = File.ReadAllText(outputPath);
+
+            this.testParser = new MLPerfMetricsParser(this.rawText, false);
             IList<Metric> metrics = this.testParser.Parse();
-            Assert.AreEqual(0, metrics.Count);
+
+            Assert.AreEqual(2, metrics.Count);
+            MetricAssert.Exists(metrics, "DGX-A100_A100-SXM4-40GBx8_TRT-custom_k_99_MaxP-SingleStream-PerformanceMode", 0, "VALID/INVALID");
+            MetricAssert.Exists(metrics, "DGX-A100_A100-SXM4-40GBx8_TRT-custom_k_99_MaxP-SingleStream-result_90.00_percentile_latency_ns", 1924537);
```

Comment:
let's make the metric name more easy to understand. For example, latency_ns_p99 as the name and then all the rest as scenario name or argument

Reply:
Using the following metric names now:

- **Performance mode**
  - PerformanceMode_p99
  - samples_per_second_p99
  - latency_ns_p99

- **Accuracy Mode**
  - AccuracyMode_p99
  - ThresholdValue_p99
  - AccuracyValue_p99
  - AccuracyThresholdRatio_p99

Some more information (config, scenario, benchmark) are logged in metadata.